oopgrade/oopgrade.py

```diff
@@ -735,6 +736,17 @@ def update_module_names(cr, namespec):
                  "WHERE module = %s ")
         logged_query(cr, query, (new_name, old_name))
 
+def update_module_from_model_data(cr, target_module, list_of_records):
+    """
+    Updates the specified models with a new module name
+
+    :param target_module: name of the module wanted to set in the models
+    :param list_of_records: list of models wanted to update with the new module
+    """
+    for record in list_of_records:
+        query = ("UPDATE ir_model_data SET module = %s "
+                 "WHERE name = %s")
```

Comment:
```suggestion
                 "WHERE name = %(record)s and module = %(source_module)s" )
```

Fer-ho així i passes un diccionari amb els parametres { 'source_module': source_module, ..}